### ISSUE TITLE
Little hammer restrictions for protected blocks/areas

### ISSUE THREAD

**Issue by LemADEC** [CreativeMD/LittleTiles#609]
As of CreativeCore_v1.10.45_mc1.12.2 + LittleTiles_v1.5.0-pre324_mc1.12.2, while in survival with the configuration set to ```"harvestLevelBlock": 1``` , I can edit blocks which are set to "pickaxe" harvest level 2. I can both cut into them and place back into them.

In practice, I would like to prevent players from using the Little Hammer to mine ores without the right tool, or breaking into protected areas by carving doors in blast proof blocks or such.
In other words, I'd like to prevent carving into non-tile blocks with hardness set to 25 or higher. I'm fine with using those blocks in structures and placing them. It's just the carving into existing non-tile block that's bothering us here.

**Comment by CreativeMD** [comment]
Just set the harvestLevel to 0 this will make it only mine stone and coal. You can also limit the number of affected vanilla blocks in the recent versions.

**Comment by CreativeMD** [comment]
I will close this issue for now, as it worked totally fine for me.

**Comment by LemADEC** [comment]
Harvest level = 0 has no effect for me. I can chisel and pickup Tin ore blocks with pickaxe harvest level iron:
![image](https://user-images.githubusercontent.com/4776032/118146722-2ef13180-b40f-11eb-8a17-959470b14716.png)
```
  "survival": {
    "limitAffectedBlocks": true,
    "maxAffectedBlocks": 2,
    "harvestLevelBlock": 0,
    "editUnbreakable": false,
    "minimumTransparency": 255,
    "limitEditBlocks": true,
    "maxEditBlocks": 10,
    "limitPlaceBlocks": true,
    "maxPlaceBlocks": 10
  },
```
Same goes with vanilla Diamond ore and obsidian.
This is reproduced in single player with only LittleTiles_v1.5.8_mc1.12.2 + CreativeCore_v1.10.58_mc1.12.2.
